src/device.c

```diff
@@ -1025,8 +1064,11 @@ uvc_error_t uvc_release_if(uvc_device_handle_t *devh, int idx) {
 
   if (UVC_SUCCESS == ret) {
     devh->claimed &= ~( 1 << idx );
-    /* Reattach any kernel drivers that were disabled when we claimed this interface */
-    ret = libusb_attach_kernel_driver(devh->usb_devh, idx);
+    if (devh->detached) {
+      /* Reattach any kernel drivers that were disabled when we claimed this interface */
+      ret = libusb_attach_kernel_driver(devh->usb_devh, idx);
+      devh->detached = 0;
+    }
```

Comment:
looks like `libusb_attach_kernel_driver` has a per-interface API,
probably `devh->detached` should be a bit-field, same ways as `devh->claimed`

Reply:
+